### DISCUSSION TITLE
Make background around canvas configurable

### DISCUSSION THREAD

**Discussion by novhack** [Orama-Interactive/Pixelorama#586] [Ideas]
Currently the background around the canvas is set by a theme. I think it would be nice to be able to set the color in preferences.
There could be a checkbox "use canvas background color from theme" to set the color back to default theme color (it could be checked by default).

**Comment by Variable-ind** [comment]
I just made it, you could check it out here: https://github.com/Variable-ind/Pixelorama/tree/Make_background_around_canvas

https://user-images.githubusercontent.com/77773850/131216263-17299a28-ffda-4ee9-a86b-08e505b8505f.mp4

**Comment by Variable-ind** [comment]
Also, i was thinking if this could be moved to "Discussions"

**Comment by OverloadedOrama** [comment]
Implemented in 66857f5e623b6247efd16b3b76e47911b128652e. I placed the option under Canvas because it seems more relevant there, but we could place it under Interface is desired.
